jina/drivers/__init__.py

```diff
@@ -200,6 +200,13 @@ def queryset(self) -> 'QueryLangSet':
         else:
             return []
 
+    @property
+    def docs(self):
+        if self.expect_parts > 1:
+            return (d for r in reversed(self.partial_reqs) for d in r.docs)
```

Comment:
@JoanFM do we reverse the requests in order to wait for the last one to arrive?

Reply:
it was like this, so to be honest not sure why it was there in the first place